plugins/providers/virtualbox/provider.rb

```diff
@@ -23,7 +23,8 @@ def self.usable?(raise_error=false)
       rescue Vagrant::Errors::VirtualBoxInvalidVersion,
              Vagrant::Errors::VirtualBoxNotDetected,
              Vagrant::Errors::VirtualBoxKernelModuleNotLoaded,
-             Vagrant::Errors::VirtualBoxInstallIncomplete
+             Vagrant::Errors::VirtualBoxInstallIncomplete,
+             Vagrant::Errors::VBoxManageNotFoundError
```

Comment:
This looks great 🎉 
Could you also add a test for this. The tests for this `usable?` method exist in [test/unit/plugins/providers/virtualbox/provider_test.rb](https://github.com/hashicorp/vagrant/blob/master/test/unit/plugins/providers/virtualbox/provider_test.rb#L27). There are some pretty solid [examples]https://github.com/hashicorp/vagrant/blob/master/test/unit/plugins/providers/virtualbox/provider_test.rb#L35) of what the test should do.

Reply:
Will do!

Reply:
Done. Test scenario is added.